src/main.cpp

```diff
@@ -157,7 +157,7 @@ void setup()
 #endif
 
 #ifdef DEBUG_PORT
-    if (!config.device.serial_disabled) {
+    if (config.device.serial_enabled) {
```

Comment:
This one may be a problem since I believe it happens before nodedb sets up all of the defaults

Reply:
Can we put in a condition where it's accessed before initialised to handle unset as enabled?